bitnami/airflow/templates/config/configmap.yaml

```diff
@@ -83,8 +83,9 @@ data:
                 . /opt/bitnami/scripts/airflow-worker-env.sh
                 . /opt/bitnami/scripts/libairflowworker.sh
 
+                export AIRFLOW_CONF_FILE=/tmp/airflow.cfg
                 airflow_generate_config # Generate the config file
-                cp /opt/bitnami/airflow/airflow.cfg /opt/bitnami/airflow/k8s-executor-config/airflow.cfg
+                cp /tmp/airflow.cfg /k8s-executor-config/airflow.cfg
```

Comment:
/k8s-executor-config doesn't exist, you created a volumeMount /k8s-executor-conf

Reply:
Good catch, sorry about that.  Can I just add a new commit to fix it? Amend is not an option given the latest CHANGELOG.md commit from bitnami-bot

Reply:
Yea I think it's fine just to add additional commits, that's what people are doing in other PRs